lms/static/scripts/frontend_apps/services/client-rpc.ts

```diff
@@ -137,6 +149,12 @@ export class ClientRPC extends TinyEmitter {
       },
     );
 
+    this._server.register('openDashboard', () => {
+      if (dashboardConfig) {
+        new DashboardOpeningFormController(dashboardConfig, authToken).render();
+      }
+    });
```

Comment:
It would be helpful to log a warning here if we receive this request but the `dashboardConfig` is missing.